src/main/java/uk/gov/hmcts/darts/audio/helper/TransformedMediaHelper.java

```diff
@@ -54,14 +56,21 @@ public UUID saveToStorage(MediaRequestEntity mediaRequest, BinaryData binaryData
 
     @Transactional
     public TransformedMediaEntity createTransformedMediaEntity(MediaRequestEntity mediaRequest, String filename,
-                                                               OffsetDateTime startTime, OffsetDateTime endTime) {
+                                                               OffsetDateTime startTime, OffsetDateTime endTime,
+                                                               Long fileSize) {
+        AudioRequestOutputFormat audioRequestOutputFormat = AudioRequestOutputFormat.MP3;
+        if (mediaRequest.getRequestType().equals(DOWNLOAD)) {
+            audioRequestOutputFormat = AudioRequestOutputFormat.ZIP;
+        }
         TransformedMediaEntity entity = new TransformedMediaEntity();
         entity.setMediaRequest(mediaRequest);
         entity.setOutputFilename(filename);
         entity.setStartTime(startTime);
         entity.setEndTime(endTime);
         entity.setCreatedBy(mediaRequest.getCreatedBy());
         entity.setLastModifiedBy(mediaRequest.getCreatedBy());
+        entity.setOutputFilesize(fileSize.intValue());
+        entity.setOutputFormat(audioRequestOutputFormat.getExtension());
```

Comment:
Worth clarifying this column with @hemantasharma1129... Perhaps it might be expected to be just MP3 or ZIP values. In which case the entity can field can be updated to be `@Enumerated(EnumType.STRING)`?

I find it all quite confusing, but I think the addAudio from darts-gateway (DAR) sends a request body with `<mediaformat>mpeg2</mediaformat>`.  We use this to save to Table "darts.media" `media_format`, that's why I added to our application properties recently: `darts.audio.allowed-media-formats`.

I know we decided recently that it made sense to add the extension to `enum AudioRequestOutputFormat` for use in the Table "darts.transformed_media" `output_filename` as the outputFilename looked wrong for download.

Reply:
i will mention to Hemanta. There's another issue I discussed re: filename - since we are now storing format in separate column, do we need it in filename column (ie extension). He says there's another ticket on it's way to sort this out.

Reply:
Yep, I wasn't sure if having the extension was important for the v2 audio-requests view `transformed_media_details[*].transformed_media_filename` e.g. T20231009-1_04_Jan_2024_1.zip

Reply:
Changed field to enum as per response from Hemanta